shared/templates/template_OVAL_yamlfile_value

```diff
@@ -51,7 +51,7 @@
   <ind:yamlfilecontent_state id="state_{{{ rule_id }}}" version="1">
     <ind:value datatype="record"{{% if ENTITY_CHECK %}} entity_check="{{{ ENTITY_CHECK }}}"{{% endif %}}>
       {{% for val in VALUES %}}
-      <field {{{ {'name': key if key in val else "#", 'datatype': val.type,  'operation':  val.operation}|xmlattr }}}>{{{ val.value }}}</field>
+      <field {{{ {'name': val.key|default("#")|escape_yaml_key, 'datatype': val.type,  'operation':  val.operation}|xmlattr }}}>{{{ val.value }}}</field>
```

Comment:
@evgenyz This change is a bugfix + enhancement, right?

Reply:
Yes